report_templates/ansible_inventory.erb

```diff
@@ -0,0 +1,144 @@
+<%#
+name: Ansible Inventory
+description: This template has a special meaning and should not be renamed.
+snippet: false
+template_inputs:
+- name: Organization
+  required: true
+  input_type: user
+  description: Whether should be Organization included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Location
+  required: true
+  input_type: user
+  description: Whether should be Location included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Host Group
+  required: true
+  input_type: user
+  description: Whether should be Host Group included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Host Collections
+  required: true
+  input_type: user
+  description: Whether should be Host Collections included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: IPv4
+  required: true
+  input_type: user
+  description: Whether should be IPv4 included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: IPv6
+  required: true
+  input_type: user
+  description: Whether should be IPv6 included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
+- name: Subnet
+  required: true
+  input_type: user
+  description: Whether should be Subnet included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Smart Proxies
+  required: true
+  input_type: user
+  description: Whether should be Smart Proxies included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Content Attributes
+  required: true
+  input_type: user
+  description: Whether should be Content Attributes included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
+- name: Facts
+  required: true
+  input_type: user
+  description: Whether should be Facts included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
+model: ReportTemplate
+require:
+- plugin: foreman_ansible
+  version: '>= 3.1.0'
+- plugin: katello
+￼  version: '>= 3.9.0'
+-%>
+<%- content_attrs = input('Content Attributes') == 'yes' %>
+<%- load_hosts(includes: [:organization, :location, :hostgroup, :host_collections, { interfaces: :subnet }, :content_facet]).each_record do |host| -%>
```

Comment:
it would be good to add hosts filter, similar to https://github.com/theforeman/community-templates/blob/develop/report_templates/host_statuses.erb#L15 (see search: `input('hosts')`, that should address @sjha4 comment about filtering